src/lib/parse/forbiddenChars.cpp

```diff
@@ -44,7 +44,7 @@ inline static bool commonForbidden(char c)
   case '>':
   case '"':
   case '\'':
-  case '=':
+//  case '=':
```

Comment:
It's very strange, because I never meant to update this pull request... All I did was making some experiments in my own fork and push it to my repo. How did it land here?